build/tools/avalanche-bridge-faq.md

```diff
@@ -0,0 +1,145 @@
+# Bridge
+
+## Transactions
+
+### What can I do if my transaction seems stuck?
+If your Ethereum transaction transferring funds over the bridge to Avalanche seems stuck and does not have any confirmations, you can speed up the transaction, as outlined below in [this](https://docs.avax.network/build/tools/avalanche-bridge-faq#can-I-safely-speed-up-my-transaction) and [this](https://docs.avax.network/build/tools/avalanche-bridge-faq#what-happens-after-I-have-sped-up-a-transaction-on-metamask). If the Ethereum transaction has already received 35 confirmations, but the Avalanche transaction timer seems to be hanging, check your Metamask wallet balance on the Avalanche network. It is likely that the transaction was already processed, but failed to be picked up by the UI due to refreshing, our speeding up the original Ethereum transaction.
```

Comment:
Not 100% sure of the internal links within this new doc are correct, but it is easy to fix once the doc is published on gitbook.